docs/development/workflow/get_devel_version.rst

```diff
@@ -229,6 +229,16 @@ ahead to the more sophisticated method look at :ref:`virtual_envs`).
     + You use Python 3.
     + You want to work on C or Cython code in `Astropy`_.
 
+Now is the time to check that you have all the required dependencies installed on 
+your machine to make sure the pip install would be a successfully one. Namely, as 
+you are effectively building from source, please check that you have both "cython"
+as well as "jinja2" installed using pip. (Note: Using conda to installed these two 
```

Comment:
I think I would omit "using pip". E.g., on my Debian system I install these using `apt-get` or `aptitude`.

Reply:
I suppose these are the borderline packages. For anything python I feel it's safe to suggest to use a python package manager rather than the distro one. That would naturally mean a full control over everything, as here we can't say "for debian" as e.g. you can't apt-get when you don't have root access, can you?

Reply:
I only meant to leave it open at this location to the reader what to use. In the next sentence, we explain our default recommendation how to do it using pip.

Reply:
I like that approach, thanks for the clarification.

Reply:
How about me specifying at the beginning of the proposed passage to be added that:

> If you have decided to use the recommended "activation" method with pip, please note the following:

Does this seem to clarify things up a bit?